common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -98,7 +98,12 @@ class InheritanceMixin(XBlockMixin):
     )
     rerandomize = String(
         display_name=_("Randomization"),
-        help=_("Specify how often variable values in a problem are randomized when a student loads the problem. Valid values are \"always\", \"onreset\", \"never\", and \"per_student\". This setting only applies to problems that have randomly generated numeric values."),
+        help=_(
+            'Specify the default for how often variable values in a problem are randomized. '
+            'This setting should be set to \"never\" unless you plan to provide a Python '
+            'script to identify and randomize values in most of the problems in your course. '
```

Comment:
How about this: 
"Specify the default value for the **Randomization** setting for individual problems. The randomization setting controls how often new values are provided in a problem with randomized variables. 
Most courses should set this default value to "Never". Use the other options only if a majority of the numerical input problems in your course include a Python script to generate random numbers."